src/mainScreen.js

```diff
@@ -79,10 +82,21 @@ export default class MainScreen extends Component {
         }
     });
   }
-
+  handlePress(){
+    return;
+  }
   render() {
     return (
       <View style={styles.container}>
+        <View style={styles.buttons}>
+          <Button
+            title="Actual score"
+            containerStyle={{padding:10, height:45, overflow:'hidden', borderRadius:20, backgroundColor: 'white'}}
```

Comment:
This style is not changed, so it should probably be created under StyleSheet.create({...